src/pages/watch later/watchLater.module.css

```diff
@@ -0,0 +1,33 @@
+.watchlater-container {
+  display: grid;
+  grid-template-columns: 10rem 1fr;
+  grid-template-rows: 1fr;
+}
+
+aside {
```

Comment:
# 
  -
    - good Naming convention in css
    - clean code,nice folder structuring
    - suggestion: you can give a foldername like side_navabar etc.